commands/command_pointer.go

```diff
@@ -26,6 +28,33 @@ func pointerCommand(cmd *cobra.Command, args []string) {
 	buildOid := ""
 	compareOid := ""
 
+	if pointerCheck {
+		var r io.ReadCloser
+		var err error
+
+		if len(pointerCompare) > 0 {
+			ExitWithError(fmt.Errorf("fatal: cannot combine --check with --compare"))
+		}
+
+		if len(pointerFile) > 0 {
```

Comment:
It looks like if a user passes in both `--pointer` and `--stdin` the check will automatically default to the pointer file.

I'm wondering if instead we should error if both flags are provided? Given that absent an error this command just returns an exit status, I could see somebody getting confused about what exactly is being checked in this case.

Reply:
This idea is great, too. These flags both exist for archaic reasons, but don't make sense when combined together with `--check`. I think that we should exit with a non-zero code and print a helpful message (as well as document this in `git-lfs-pointer.1.ronn`, and test it in `t/`, too)